tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -259,6 +277,7 @@ - (void)addParentTo:(NSProgress*)child {
 }
 
 ARM_DISABLED_TEST(NSProgress, SingleParentEvenAcrossThreads) {
+    #if TARGET_OS_WIN32
```

Comment:
This has to do with the     } @catch (NSException* exception) {
        _exceptionWasCaught = YES;
    }
}

given that's pretty much amounts to what the test is.
I'll leave it as is.

---
In reply to: [73764878](https://github.com/Microsoft/WinObjC/pull/836#discussion_r73764878) [](ancestors = 73764878,73764865)